lib/did-io.js

```diff
@@ -194,22 +194,34 @@ export function _methodById({doc, methodId}) {
 }
 
 /**
- * Parses the DID into various component (currently, only cares about prefix).
+ * Parses the DID into scheme and method component.
  *
  * @example
  * parseDid({did: 'did:v1:test:nym'});
- * // -> {prefix: 'v1'}
+ * // -> {method: 'v1'}
```

Comment:
It's returning the scheme too.

Reply:
Addressed here and good catch: https://github.com/digitalbazaar/did-io/pull/63/commits/d97608615ee9df8be573fdaa797a436379a699a2